src/main.ts

```diff
@@ -49,4 +49,4 @@ async function bootstrap() {
   logger.log(`Listening on port ${appConfig.port}`, 'AppBootstrap');
 }
 
-bootstrap();
+void bootstrap();
```

Comment:
I think the better solution is to add the return type `Promise<void>` to the function.

Reply:
That won't help here. [no-floating-promises](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-floating-promises.md) wants us to handle all promises or explicitly ignore them with `void`.